corehq/util/couchdb_management.py

```diff
@@ -36,6 +36,12 @@ def all_dbs_by_slug(self):
         return {slug: Database(db_uri)
                 for slug, db_uri in self.all_db_uris_by_slug.items()}
 
+    @property
+    @memoized
+    def all_dbs_by_name(self):
+        return {Database(db_uri).dbname: Database(db_uri)
+                for db_uri in self.all_db_uris_by_slug.values()}
+
     def get_db(self, postfix):
```

Comment:
Looking at this I should probably have called the parameter `slug` instead of `postfix` at least, if not changed the function name to `get_db_for_slug`